### DISCUSSION TITLE
Install dotfiles on fedora

### DISCUSSION THREAD

**Discussion by ShinVN** [end-4/dots-hyprland#671] [Extra Distros]
i need help installing this dotfile on fedora

**Comment by end-4** [comment]
I'm not guaranteeing anything will work, but see this https://end-4.github.io/dots-hyprland-wiki/en/i-i/01setup/#manual-installation-basically-any-distro

**Comment by pizzav-xyz** [comment]
@ShinVN if this worked can you send the package list

**Reply by ShinVN** [reply]
ok :"D

**Reply by pizzav-xyz** [reply]
> ok :"D

its been a bit, could you provide an update?

**Comment by clsty** [comment]
Update: since #2393, now you can use `./setup install` for Fedora.

See also [dist-fedora/README.md](https://github.com/end-4/dots-hyprland/blob/241f33fb2fbbdfe0dac4dcf7e7d1dc9aa8d3129f/sdata/dist-fedora/README.md).
